runtime/relay-encoder/src/kusama.rs

```diff
@@ -192,657 +192,657 @@ pub const KUSAMA_RELAY_INDICES: RelayChainIndices = RelayChainIndices {
 	close_channel: 2u8,
 	cancel_open_request: 6u8,
 };
-
-#[cfg(test)]
-mod tests {
-	use super::*;
-	use crate::kusama::KusamaEncoder;
-	use frame_support::traits::PalletInfo;
-	use sp_runtime::Perbill;
-	use xcm_primitives::{StakeEncodeCall, UtilityEncodeCall};
-
-	#[test]
-	fn test_as_derivative() {
```

Comment:
Is there a specific reason I'm missing of why all these tests were removed?

Reply:
kusama  runtime has been removed from the the polkadot-sdk and we are considering other ways  to perform those tests without relaying on the kusama runtime